src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php

```diff
@@ -182,4 +194,38 @@ public function shutdownHandler($signo = null)
         ));
         return true;
     }
+
+    /**
+     * Event triggered on Worker.job.empty
+     *
+     * @param Event $event
+     * @return void
+     */
+    public function jobEmpty(Event $event)
```

Comment:
You can just add `$event` to the first line of each method to ensure it is "used" by phpcs.